File: ram.py
from perangkat import Perangkat

class RAM(Perangkat):
    def __init__(self, merk, nama, kapasitas_gb, ddr):
        super().__init__(merk, nama)
        self.kapasitas_gb = kapasitas_gb
        self.ddr = ddr
    
    def info(self):
        return f"RAM: {self.merk} {self.nama} ({self.kapasitas_gb}GB, {self.ddr})"

File: ram.cpp
#include "ram.h"

RAM::RAM(std::string merk, std::string nama, int kapasitas_gb, std::string ddr)
    : Perangkat(merk, nama), kapasitas_gb(kapasitas_gb), ddr(ddr) {}

std::string RAM::info() const {
    return "RAM: " + merk + " " + nama + " (" + std::to_string(kapasitas_gb) + "GB, " + ddr + ")";
}
